### ISSUE TITLE
Enable setting filter frequency in hertz and notes

### ISSUE THREAD

**Issue by ianhattwick** [gibber-cc/gibber#88]
Normalizing the filter frequency to (0, 1) doesn't make sense for choosing appropriate filter settings. . .

**Comment by charlieroberts** [comment]
The final cutoff frequency used in synths for a given sample is determined by the cutoff value, the note being played, the envelope of the synth, and the `filterMult` property. So selecting an actual frequency is somewhat meaningless given the combination of all these parameters. I guess I could make the actual equation used more explicit in the docs?

**Comment by charlieroberts** [comment]
Here's the [relevant code for the `Synth` instrument](https://github.com/gibber-cc/gibberish/blob/ec6d062a48455b7e0b3c3a25d3384f8ca456e278/js/instruments/synth.dsp.js#L46). I forgot that the global `loudness` property of the synth also plays a role in the filter cutoff, as well as the loudness of individual notes that are triggered (this is `triggerLoudness` in the code). So many elements control the final cutoff frequency!

**Comment by charlieroberts** [comment]
oh wait, are you talking about in standalone filters? hmmm. that could make more sense.

**Comment by ianhattwick** [comment]
Yes - I was thinking of standalone filters. For synths I agree that something normalized makes sense as it will probably track VCO frequency.
